django/utils/autoreload.py

```diff
@@ -269,7 +272,12 @@ def run(self, django_main_thread):
         from django.urls import get_resolver
         # Prevent a race condition where URL modules aren't loaded when the
         # reloader starts by accessing the urlconf_module property.
-        get_resolver().urlconf_module
+        try:
+            get_resolver().urlconf_module
+        except Exception:
+            # Loading the urlconf can result in errors during development.
+            # If this occurs then swallow the error and continue.
+            pass
```

Comment:
Can we catch specific exceptions?

Reply:
I don’t think so, we need to catch anything that could be thrown (syntax errors, runtime errors, etc etc). These are bubbled up during an import